lib/app/data/models/quote_model.dart

```diff
@@ -0,0 +1,21 @@
+class Quote {
+  String quote;
+  String author;
+
+  Quote({
+    required this.quote,
+    required this.author,
+  });
+
+  String getQuote() {
+    return quote;
+  }
+
+  String getAuthor() {
+    return author;
+  }
+
+  Quote.fromJson(Map json)
```

Comment:
This is not actually right - you're parsing a map and not json right? Let's name it as fromMap then

Reply:
Yes, sorry I missed that.